proto/pdpb.proto

```diff
@@ -295,3 +295,35 @@ message Error {
     optional BootstrappedError     bootstrapped = 2;
     optional StoreIsTombstoneError is_tombstone = 3;
 }
+
+service PlacementService {
+    rpc Tso(Request) returns (Response) {}
```

Comment:
> I prefer TsoRequest

> How about making another TsoRequest2 { RequestHeader, TsoRequest }?

Since both of those require a **large** change and have some compatibility issues, I suggest we should define a new _pdpb2.proto_ from the ground up, protobuf syntax = "proto3".